test/node/utils/interaction-utils-test.js

```diff
@@ -131,7 +131,7 @@ test('interactionUtil -> autoFindTooltipFields', t => {
   };
 
   t.deepEqual(
-    findFieldsToShow({fields, id: 'test'}),
+    findFieldsToShow({fields, id: 'test', maxDefaultTooltips: 5}),
```

Comment:
I would change the value here to be `maxDefaultTooltips: 3` to test wether the custom value is passed in

Reply:
Good point! Will fix this.